test_scripts/Defects/5_0/1831_Crash_during_Navigation_app_resume.lua

```diff
@@ -0,0 +1,88 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/smartdevicelink/sdl_core/issues/1831
+--
+-- Description:
+-- Crash during Navigation app resume
+-- Precondition:
+-- Core and HMI are started.
+-- Navigation application is registered and activated. -> HMI level = FULL
+-- In case:
+-- 1) BC.ActivateApp is sent from SDL to HMI
+-- 2) Once the HMI gives the response to the BC.ActivateApp,
+--    then there is a crash observed in state_controller_impl.cc where below mentioned DCHECK is failing.
+-- Expected result:
+-- 1) SDL must resume Navigation app
+-- Actual result:
+-- Crash during Navigation app resume
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local common = require('user_modules/sequences/actions')
+local runner = require('user_modules/script_runner')
+local test = require("user_modules/dummy_connecttest")
+local mobile_session = require('mobile_session')
+local commonTestCases = require('user_modules/shared_testcases/commonTestCases')
+local utils = require("user_modules/utils")
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ General configuration parameters ]]
+config.defaultProtocolVersion = 2
+
+--[[ Local Variables ]]
+config.application1.registerAppInterfaceParams.appHMIType = { "NAVIGATION" }
+config.application1.registerAppInterfaceParams.isMediaApplication = true
+
+--[[ Local Functions ]]
+local function closeSession()
```

Comment:
in function is performed the connection closing, not session, please update function name